Source/WebCore/page/scrolling/mac/ScrollerMac.mm

```diff
@@ -370,6 +371,7 @@ - (void)invalidate
 void ScrollerMac::updateScrollbarStyle()
 {
     m_scrollerImp = [NSScrollerImp scrollerImpWithStyle:nsScrollerStyle(m_pair.scrollbarStyle()) controlSize:nsControlSizeFromScrollbarWidth(m_pair.scrollbarWidthStyle()) horizontal:m_orientation == ScrollbarOrientation::Horizontal replacingScrollerImp:takeScrollerImp().get()];
```

Comment:
If we called `setScrollerImp()` here that would take care of the `updateMinimumKnobLength() call.

Not sure how we get away with not setting the delegate like we did above?

Reply:
Fixed. The delegate gets passed over through `replacingScrollerImp`.